nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/ConsumeJMS.java

```diff
@@ -86,6 +86,29 @@
             .defaultValue(CLIENT_ACK.getValue())
             .build();
 
+    static final PropertyDescriptor DURABLE_SUBSCRIBER = new PropertyDescriptor.Builder()
+                                                             .name("Durable subscription")
+                                                             .description("If destination is Topic if present then make it the consumer durable.")
+                                                             .required(false)
+                                                             .expressionLanguageSupported(true)
+                                                             .defaultValue("true")
+                                                             .allowableValues("true", "false")
+                                                             .build();
+    static final PropertyDescriptor SHARED_SUBSCRIBER = new PropertyDescriptor.Builder()
+                                                            .name("Shared subscription")
+                                                            .description("TIf destination is Topic if present then make it the consumer durable.")
```

Comment:
I found a typo here. Is it possible to add more details on how this configuration change consumer behavior? Something like:

> Specify whether to share the subscription, only effective If destination is a Topic. If enabled, messages can be distributed among NiFi nodes in a cluster. If not, each NiFi node will work as an individual consumer and receive the same messages.

Is my understanding above correct??